Furball.Vixie/Graphics/Backends/Veldrid/FullScreenQuadShaders.cs

```diff
@@ -0,0 +1,47 @@
+namespace Furball.Vixie.Graphics.Backends.Veldrid {
+    internal class FullScreenQuadShaders {
+        public static string VertexShader = @"#version 450
```

Comment:
is there anything preventing this from being in ShaderCode/Veldrid/? I'd much rather have that there

Reply:
> is there anything preventing this from being in ShaderCode/Veldrid/? I'd much rather have that there

convenience mainly lol, it was easy to just put it in a local class then it was to deal with embedded resource loading, but ill move it